cmd/skaffold/app/cmd/config/util.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/kubectl"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func resolveKubectlContext() {
+	if kubectx != "" {
+		return
+	}
+	var buf bytes.Buffer
+	w := bufio.NewWriter(&buf)
+	k := kubectl.CLI{}
+
+	if err := k.Run(nil, w, "config", nil, "current-context"); err != nil {
```

Comment:
we already have code that does this through the client library

Reply:
https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/kubernetes/context/context.go

Reply:
its also memoized

Reply:
oh cool, I'll use this